internal/vpn/client.go

```diff
@@ -188,16 +182,32 @@ func (c *Client) Serve() error {
 
 	c.setAppStatus(ClientStatusConnecting)
 
-	r := netutil.NewDefaultRetrier(c.log)
+	errTransportNotFound := appserver.RPCErr{
+		Err: rfclient.ErrTransportNotFound.Error(),
+	}
+
+	r := netutil.NewRetrier(c.log, netutil.DefaultInitBackoff, netutil.DefaultMaxBackoff, 3, netutil.DefaultFactor).
+		WithErrWhitelist(errHandshakeStatusForbidden).WithErrWhitelist(errHandshakeStatusInternalError).
+		WithErrWhitelist(errHandshakeNoFreeIPs).WithErrWhitelist(errHandshakeStatusBadRequest).
+		WithErrWhitelist(errTransportNotFound)
```

Comment:
`WithErrWhitelist` accept multiple error, so why didn't add all once and use multiple `WithErrWhitelist` method? Something like this:
```Go
r := netutil.NewRetrier(c.log, netutil.DefaultInitBackoff, netutil.DefaultMaxBackoff, 3, netutil.DefaultFactor).
	WithErrWhitelist(
		errHandshakeStatusForbidden,
		errHandshakeStatusInternalError,
		errHandshakeNoFreeIPs,
		errHandshakeStatusBadRequest,
		errTransportNotFound,
	)
```

Reply:
That was an oversight on my part. Thanks for pointing that out.